docs/DockerContainerLifecycle.md

```diff
@@ -0,0 +1,203 @@
+<!--- master only -->
+> ![warning](images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.9.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Lifecycle of a container (Calico with no Docker networking)
+
+This page provides low-level details about what happens within Calico when 
+Docker containers are networked with Calico when Docker is not providing any
+networking.
+
+> The main features covered in this document apply to the situation 
+> when Calico is implemented as a Docker network plugin, however, in this case
+> many steps are handled automatically by Dockers networking framework and the
+> Calico plugin - this will be covered in a future document.
+ 
+We will start with two hosts and end up with containers on each host that have
+connectivity between them.
+
+This walk-through includes details about user actions, the data added to the
+etcd datastore and the response of the components that make up the calico-node
+container which provides the Calico networking service.
+
+This article has the following sub-sections, each representing a particular
+user action:
+  - Start Calico on two hosts
+  - Create and add containers to Calico
+  - Configure endpoints for each container
+  - Create a Calico policy profile
+  - Associate the Calico endpoints with the profile
+
+## 1. Start Calico on two hosts
+
+User launches the Calico service by running 
+
+    calicoctl node
+    
+on both hosts (Host1 and Host2).
+
+### System response
+
+The calicoctl utility:
+
+-  Launches a container called `calico-node` on both hosts. The container is
+   created from the `calico/node` Docker image.
+-  Auto-detects the hosts IP and configures the IP as the BGP address in etcd.
+-  Ensures that there are a number of global default values configured in etcd.
+   This includes:
+   -  The default AS number to use for BGP
+   -  Initialising the BGP node-to-node mesh option
+-  Configures two default IP Pools (`192.168.0.0/16` and 
+  `fd80:24e2:f998:72d6::/64`)
+   
+Inside the calico-node container:
+
+-  Three processes are started:  Felix, confd and Bird (see 
```

Comment:
Strictly speaking it's at least 4 - there's bird6 too

I think there's also the log filters but I'm fine not mentioning them